Multisig/UI/Transaction/TransactionListViewController/TransactionListViewController.swift

```diff
@@ -388,9 +388,9 @@ class TransactionListViewController: LoadableViewController, UITableViewDelegate
             decimals = 0
         case .ether(let etherTransferInfo):
```

Comment:
should we also rename `.ether`?

Reply:
This is a separate question, because this one is the txInfo and it depends on the client gateway.